modules/on_gke/main.tf

```diff
@@ -14,61 +14,149 @@
  * limitations under the License.
  */
 
-//*****************************************
-//  Create Keys for Forseti server and
-//  client service accounts.
-//*****************************************
+resource "random_id" "random_hash_suffix" {
+  byte_length = 4
+}
+
+resource "random_integer" "random_minute" {
+  min = 0
+  max = 59
+}
+
+resource "null_resource" "org_id_and_folder_id_are_both_empty" {
+  count = length(var.composite_root_resources) == 0 && var.org_id == "" && var.folder_id == "" ? 1 : 0
+
+  provisioner "local-exec" {
+    command     = "echo 'composite_root_resources=${var.composite_root_resources} org_id=${var.org_id} folder_id=${var.org_id}' >&2; false"
+    interpreter = ["bash", "-c"]
+  }
+}
 
-resource "google_service_account_key" "server_key" {
-  service_account_id = var.forseti_server_service_account
+resource "null_resource" "email_without_sendgrid_api_key" {
+  count = var.inventory_email_summary_enabled == "true" && var.sendgrid_api_key == "" ? 1 : 0
```

Comment:
Should "true" be bool? Variable here is bool: https://github.com/forseti-security/terraform-google-forseti/blob/1d2a79c3d43b6b285863dcc7595e8f0a1ec14711/modules/on_gke/variables.tf#L667